helix-term/src/job.rs

```diff
@@ -33,7 +33,7 @@ impl Job {
         f: F,
     ) -> Job {
         Job {
-            future: f.map(|r| r.map(|x| Some(x))).boxed(),
+            future: f.map(|r| r.map(Some)).boxed(),
```

Comment:
Is this a result? If so can just `r.ok()`.

Reply:
Ah wait, no, I'm going from `Result<T>` to `Result<Option<T>>`, not `Option<T>`.